packages/extended-text-entry/src/main.jsx

```diff
@@ -1,24 +1,37 @@
-import React from "react";
-import TextArea from "./textarea";
+import React from 'react';
+import EditableHTML from '@pie-lib/editable-html';
+import PropTypes from 'prop-types';
 
 class Main extends React.Component {
-  constructor (props) {
+
+  static propTypes = {
+    onChange: PropTypes.func.isRequired,
+    model: PropTypes.object
+  }
+
+  constructor(props) {
     super(props);
+    
+    this.handleEditableChange = this.handleEditableChange.bind(this);
+  }
+
+  handleEditableChange(change) {
+    this.props.onChange(change);
   }
 
   render() {
+    let {width, height} = this.props.model;
+
     return (
-      <React.Fragment>
-        <TextArea
-          rows="8"
-          cols="70"
-          value="i am the value"
-          placeholder="Hey I am placeholder"
+      <EditableHTML
+        onChange={this.handleEditableChange}
+        markup=""
```

Comment:
you need to bind the `session.value` here no?

Reply:
yes i am doing `this.props.onChange(change);`